plugin/pick.vim

```diff
@@ -68,3 +72,16 @@ function! s:BufferListCommand()
   let buffers = map(bufnrs, 'bufname(v:val)')
   return 'echo "' . join(buffers, "\n") . '"'
 endfunction
+
+function! s:TagCommand()
+  let l:tag_file = ""
+
+  for tfile in split(&tags, ",")
+    if filereadable(expand(tfile)) == 1
+      let l:tag_file = tfile
+      break
+    endif
+  endfor
+
+  return "awk -F$'\t' '{print $1}' " . l:tag_file . "  | sort -u | grep -v '^!'"
```

Comment:
Could we support multiple tag files by replacing unescaped commas (not preceded by a backslash) with spaces and passing the resulting string directly to awk?